chapters/modules/chapter_1/pages/environment_setup.adoc

```diff
@@ -1,7 +1,162 @@
+pyenv activate cairo_venv
+----
+
+Either way, make sure the venv is activated – you should see (cairo_venv) in the command line prompt. 
+Your terminal should look like this:
+
+[,Bash]
+----
+(cairo_venv) satoshi@nakamoto-notebook % 
+image::cairo_venv.png[cairo venv]
+
+----
+
+=== +++<strong>+++Step 2: Install the necessary dependencies +++</strong>+++
+
+On Ubuntu, for example, you will have to first run:
+
+[,Bash]
+----
+sudo apt install -y libgmp3-dev
+----
+
+On Mac, you can use brew:
+
+[,Bash]
+----
+brew install gmp
+----
+
+Finally, install the following dependencies:
+
+[,Bash]
+----
+(cairo_venv) satoshi@nakamoto-notebook % pip install ecdsa fastecdsa sympy
+----
+
+There might be some other libraries which are not direct dependencies but they might be indirect dependencies that you would need to install based on your operating system.
+
+
+=== +++<strong>+++Step 3: Install cairo-lang CLI +++</strong>+++
+
+If you had cairo-lang installed previously, uninstall it and install the latest version. 
+
+[,Bash]
+----
+#Uninstall the previous version
+pip uninstall cairo-lang
+#Build locally
+pip install cairo-lang
+----
+
+Once you have installed the cairo lang package and installed it, make sure to test your installation by running the following command:
+
+[,Bash]
+----
+(cairo_venv) satoshi@nakamoto-notebook % starknet --version      
+starknet 0.11.0.2
+
+----
+
+Please check that the version returned matches.
+____
+NOTE: For M1 and M2 users, if you face any issues while running starknet --version command, 
+then you may refer to https://github.com/OpenZeppelin/nile/issues/22[this] article to follow alternate steps to install the dependencies in Step 2.
+____
+
+== `cairo` Cairo 1 compiler Installation
+
+=== +++<strong>+++Step1: Clone the cairo repository and set it up using the following instructions:+++</strong>+++
+
+[,Bash]
+----
+# Go to your $HOME directory
+cd ~/
+# Clone the `cairo` Cairo 1 compiler
+git clone https://github.com/starkware-libs/cairo/
+
+#Rename the cairo folder to .cairo in order to keep it hidden
+mv cairo .cairo
+
+#Checkout into the working branch and generate the release binaries
+cd .cairo/
+git checkout tags/v1.0.0-alpha.6
+cargo build --all --release
+
+----
+
+=== +++<strong>+++Step 2: Add Cairo 1.0 executables to your path:+++</strong>+++
+
+Now that we have built the Cairo 1.0 binaries, we need to add them to the PATH environment variable. Add the following in your .bashrc or .zshrc:
+
+[,Bash]
+----
+#Add the below command to your .bashrc or .zshrc
+export PATH="$HOME/.cairo/target/release:$PATH"
+----
+
+After that, open a new shell and check that the following command returns a version number:
+
+[,Bash]
+----
+cairo-compile --version
+----
+
+Your output should look something like this(not necessarily inside the cairo_venv):
+[,Bash]
+----
+(cairo_venv) satoshi@nakamoto-notebook % cairo-compile 0.11.0.2
+----
+
+image::cairo-compile.png[cairo-compile version]
```

Comment:
this image is not rendering on Github, is it rendering correctly in Antora?

Reply:
Yes, it was but I removed the images and made the outputs based on your earlier suggestion.